### ISSUE TITLE
Booleans in CODAP

### ISSUE THREAD

**Issue by retabak** [P-bibs/codap-transformers#248]
So, I'm suspecting this is a CODAP issue and not a Transformers issue but I'm not sure, so....

The original Animals Dataset spreadsheet always had `TRUE` and `FALSE` in all caps.

CODAP didn't seem to like that, so I updated the spreadsheet with `true` and `false` in lowercase.

The funny thing is, when I use `true` and `false` in expressions (for the Transformers), I still need to enter them as strings - in quotation marks. Otherwise, I get weird results. (For example, if I enter `fixed = true`, I get back an empty dataset.)

any idea what's going on? I assume this is not in our control.

**Comment by thomascastleman** [comment]
That's correct, I believe this is behavior stemming from CODAP not parsing booleans correctly when it imports a sheet. They are left as strings, which is why you're seeing the filter work with `"true"` but not `true`. I believe [they recognize this as a bug and intend to fix it](https://codap.concord.org/forums/topic/booleans-in-google-sheets-importer/) but I'm not sure what progress has been made.
